src/constraints/storage_constraints.jl

```diff
@@ -207,6 +220,17 @@ function add_cold_thermal_storage_dispatch_constraints(m, p, b; _n="")
         m[Symbol("dvStoragePower"*_n)][b] >= m[Symbol("dvDischargeFromStorage"*_n)][b,ts] + 
         sum(m[Symbol("dvProductionToStorage"*_n)][b,t,ts] for t in p.techs.cooling)
     )
+
+    #Do not allow GHP to charge storage
```

Comment:
@zolanaj  or @Bill-Becker is this assumption documented anywhere? If not, is it important to do so at some point?

Reply:
I am raising an issue [here](https://github.com/NREL/REopt.jl/issues/392) as a placeholder to allow for further discussion.